ui/src/main/webapp/tests/app/registeredapplication.model.spec.ts

```diff
@@ -0,0 +1,9 @@
+import {RegisteredApplicationModel} from "../../app/models/registered.application.model";
+
+describe('registered application model tests', () => {
+    it ('has input path', () => {
+        let model = new RegisteredApplicationModel();
+        model.inputPath = "testpath";
+        expect(model.inputPath).toEqual("testpath");
+    });
```

Comment:
Yes, that is basically true. It is just an example of a synchronous test (whereas the other one is async).